Dockerfile

```diff
@@ -1,21 +1,34 @@
-#FROM golang:1.15-buster AS builder
-#WORKDIR /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver
-#COPY . .
-#RUN make
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM --platform=$BUILDPLATFORM golang:1.17 AS builder
+WORKDIR /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver
+ADD . .
+ARG TARGETPLATFORM
+RUN ARCH=$(echo $TARGETPLATFORM | cut -f2 -d '/') make driver
 
 FROM --platform=linux/ppc64le k8s.gcr.io/build-image/debian-base:v2.1.3 AS debian-base
 RUN clean-install ca-certificates e2fsprogs mount udev util-linux xfsprogs
 RUN clean-install bash multipath-tools sg3-utils
-# COPY --from=builder /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver/bin/ibm-powervs-block-csi-driver /bin/ibm-powervs-block-csi-driver
-COPY ./bin/ibm-powervs-block-csi-driver /bin/ibm-powervs-block-csi-driver
-
-ENTRYPOINT ["/bin/ibm-powervs-block-csi-driver"]
+# Copy driver binary from builder
+COPY --from=builder /bin/ibm-powervs-block-csi-driver /ibm-powervs-block-csi-driver
+ENTRYPOINT ["/ibm-powervs-block-csi-driver"]
 
 FROM --platform=linux/ppc64le registry.access.redhat.com/ubi8/ubi:8.5 AS rhel-base
 RUN yum --disableplugin=subscription-manager -y install httpd php \
   && yum --disableplugin=subscription-manager clean all
 RUN clean-install ca-certificates e2fsprogs mount udev util-linux xfsprogs
```

Comment:
will it work? because this `clean-install` is available only in the k8s images like `k8s.gcr.io/build-image/debian-base:v2.1.3`

Reply:
rhel is currently ignored.

Reply:
Support will be added for RHEL future.

Reply:
lets have a different task for refactoring ubi(RHEL) based image